website/static/js/osfLanguage.js

```diff
@@ -80,5 +80,10 @@ module.exports = {
         dataSendError: 'Error sending data to the server: check that all fields are valid, or contact ' + SUPPORT_LINK + ' if the problem persists.',
         creationSuccess: 'Successfully registered new application',
         dataUpdated: 'Application data updated'
+    },
+    projectSettings: {
+        updateSuccessMessage: 'Category, title, and description updated successfully',
+        updateErrorMessage: 'Error updating project settings',
```

Comment:
This error should give the user an actionable way to correct the issue. Since this is displayed when an unexpected occur (500) occurs, the user should contact support.

```
Could not update project settings at this time. Please try again or contact ....
```